plugins/engines/ospray/OSPRayFrameBuffer.cpp

```diff
@@ -21,7 +21,7 @@
 #include "OSPRayFrameBuffer.h"
 
 #include <brayns/common/log.h>
-#include <ospray/common/OSPCommon.h>
+#include <SDK/common/OSPCommon.h>
```

Comment:
see above comment. Looks weird here to have top-level SDK folder. Imagine /usr/include/SDK?!? Who knows that this is ospray? :)